openapi/crm/crm.json

```diff
@@ -0,0 +1,381 @@
+{
+  "openapi": "3.1.0",
+  "x-stoplight": {
+    "id": "iyzxipdhj3szx"
+  },
+  "info": {
+    "title": "CRM",
+    "version": "1.0",
+    "description": ""
+  },
+  "servers": [
+    {
+      "url": "https://prod.apigateway.co/org",
+      "description": "Production"
+    },
+    {
+      "url": "https://demo.apigateway.co/org",
+      "description": "Demo"
+    },
+    {
+      "url": "http://localhost:11001/org",
+      "description": "Local"
+    },
+    {
+      "url": "{local}/org",
+      "description": "Local"
+    }
+  ],
+  "paths": {
+    "/{namespace}/{resourceTypeCode}/meta/fields": {
+      "parameters": [
+        {
+          "schema": {
+            "type": "string"
+          },
+          "name": "namespace",
+          "in": "path",
+          "required": true,
+          "description": "The namespace, PID/AGID"
+        },
+        {
+          "schema": {
+            "type": "string"
+          },
+          "name": "resourceTypeCode",
+          "in": "path",
+          "required": true,
+          "description": "The resourceTypeCode, here Contact"
+        }
+      ],
+      "get": {
+        "summary": "Get field schema",
+        "tags": [],
+        "responses": {
+          "200": {
+            "description": "This API follows [Open API schema](https://spec.openapis.org/oas/v3.1.0#schema-object) of structure for response.",
+            "content": {
+              "application/json": {
+                "examples": {
+                  "Get crm field": {
+                    "value": {
+                      "fields": [
+                        {
+                          "externalId": "externalId_xxxx",
```

Comment:
The ListFieldSchema has evolved and has additional fields and varied structure to the attributes. Are we looking to use the example as source of truth or just for demonstration purpose? Depending on the purpose we need to update or modify it example

Reply:
This is just for  demonstration purpose. We are not going to expose our internal fields.